tests/python/unittest/test_arith_intset.py

```diff
@@ -59,14 +64,57 @@ def test_check():
 
     # multiple compare operators
     res2 = tvm.arith.DeduceBound(a, (a+b>3)>c , {b: b_s, c: c_s}, {})
-    assert res1.is_nothing()
+    assert res2.is_nothing()
 
     # multiple target variable
     res2 = tvm.arith.DeduceBound(a, a*2-a>b, {b: b_s}, {})
-    assert res1.is_nothing()
+    assert res2.is_nothing()
+
+def test_deduce_relax():
+    def test_pos(a1, a2):
+        a = tvm.var('a')
+        b = tvm.var('b')
+        b_s = tvm.arith.intset_interval(a1, a2)
+        e0 = 2*b + a*4 + 3
+        res1 = tvm.arith.DeduceBound(a, e0<17, {b: b_s}, {b: b_s})
+        assert tvm.ir_pass.Simplify((res1.max() * 4 + 3 + 2*b_s.max()) < 17).value == 1
+
+        res1 = tvm.arith.DeduceBound(a, e0>17, {b: b_s}, {b: b_s})
+        assert (tvm.ir_pass.Simplify((res1.min() * 4 + 3 + 2*b_s.min()) > 17)).value == 1
+
+        res1 = tvm.arith.DeduceBound(a, e0<=17, {b: b_s}, {b: b_s})
+        assert (tvm.ir_pass.Simplify((res1.max() * 4 + 3 + 2*b_s.max()) <= 17)).value == 1
+      
+        res1 = tvm.arith.DeduceBound(a, e0>=17, {b: b_s}, {b: b_s})
+        assert (tvm.ir_pass.Simplify((res1.min() * 4 + 3 + 2*b_s.min()) >= 17)).value == 1
+   
+    def test_neg(a1, a2):
+        a = tvm.var('a')
+        b = tvm.var('b')
+        b_s = tvm.arith.intset_interval(a1, a2)
+        e0 = (-2) * b + a*(-4) + 3
+        res1 = tvm.arith.DeduceBound(a, e0<21, {b: b_s}, {b: b_s})
+        assert tvm.ir_pass.Simplify((res1.min() * (-4) + 3 + (-2)*b_s.max()) < 21).value == 1
+
+        res1 = tvm.arith.DeduceBound(a, e0>21, {b: b_s}, {b: b_s})
+        assert (tvm.ir_pass.Simplify((res1.max() * (-4) + 3 + (-2)*b_s.min()) > 21)).value == 1
+
+        res1 = tvm.arith.DeduceBound(a, e0<=21, {b: b_s}, {b: b_s})
+        assert (tvm.ir_pass.Simplify((res1.min() * (-4) + 3 + (-2)*b_s.max()) <= 21)).value == 1
+
+        res1 = tvm.arith.DeduceBound(a, e0>=21, {b: b_s}, {b: b_s})
+        assert (tvm.ir_pass.Simplify((res1.max() * (-4) + 3 + (-2)*b_s.min()) >= 21)).value == 1
+       
```

Comment:
add a testcase with more level of nesting, like
```
(x * b + a* 2) * 4
```

Reply:
a = tvm.var('a')
        b = tvm.var('b')
        c = tvm.var('c')
        b_s = tvm.arith.intset_interval(a1, a2)
        c_s = tvm.arith.intset_interval(10, 20)
        e0 = (b*c + a* coff) * 4

        res1 = tvm.arith.DeduceBound(a, e0<63, {b: b_s}, {b: b_s})

res1.max() = ((((63 - 1)/4) - select((c >= 0), (4*c), (0*c)))/4) when coff > 0

I guess there is work to do